.github/workflows/test.yml

```
@@ -67,3 +67,42 @@ jobs:
           FOLDER: D3DB
           SKIP_EMPTY_COMMITS: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  build-website:
+    name: Run D3 Build of website for github pages
+    environment:
+      name: d3db-gh-pages
+      url: https://github.com/TechWorksHub/ManySecured-D3DB/tree/gh-pages
+    concurrency: d3db-gh-pages # limit to a single-run build at a time
+    runs-on: ubuntu-latest
+    # d3build is slow, so only run if lint passes
+    needs: [pre-commit, d3lint]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install graphviz
+        run:  sudo apt-get update && sudo apt install graphviz
```

You may want to use the [actions/setup-python](https://github.com/actions/setup-python) action to ensure that you have a specific version of Python installed, and to set up caching automatically.

```suggestion
        run:  sudo apt-get update && sudo apt install graphviz
      - uses: actions/setup-python@v4
        with:
          python-version: '3.9'
          cache: 'pip' # caching pip dependencies
```